libcontainer/container_linux.go

```diff
@@ -1504,11 +1504,14 @@ func (c *linuxContainer) criuSwrk(process *Process, req *criurpc.CriuReq, opts *
 	// cmd.Process will be replaced by a restored init.
 	criuProcess := cmd.Process
 
+	var st *os.ProcessState
```

Comment:
nit: this name is good when is used in a block of 20-lines code, but in this case, I would use something more descriptive (e g criuProcessState).

Reply:
Your comment has been addressed.